### DISCUSSION TITLE
Quickstart page

### DISCUSSION THREAD

**Discussion by steto123** [pyscf/pyscf#2514] [General]
I am now very unsure where to post my problems. I'm working my way through the documentation because I want to couple pyscf with rdkit and tensorflow2. And now, while systematically working through the examples on the quickstart page, I came across the following problem (all examples on the quickstart page are in a Jupyter notebook and work very well up to that point). The example given in the section “spatially localized orbitals” - https://pyscf.org/quickstart.html#spatially-localized-molecular-orbitals - fails.  The following error is displayed:
TypeError Traceback (most recent call last)
Cell In[12], line 5
      3 from pyscf import lo
      4 occ_orbs = rhf_h2o.mo_coeff[:, rhf_h2o.mo_occ > 0].
----> 5 fb_h2o = lo.Boys(mol_h2o, occ_orbs, rhf_h2o) # Foster-Boys
      6 loc_occ_orbs = fb.kernel()
      7 virt_orbs = rhf_h2o.mo_coeff[:, rhf_h2o.mo_occ == 0].

TypeError: OrbitalLocalizer.__init__() takes from 2 to 3 positional arguments but 4 were given

According to my rather sparse knowledge of Python, more parameters than expected are passed here, but I don't see the problem. I see 3 arguments, but jupyter notebook claims 4.  What is my problem?

After I had commented out the two lines with Foster - Boys, there was an second error in the last line of the example. I changed it as follows: 
loc_virt_orbs = pm_h2o.kernel()

This then leads to success, i.e. error-free processing.

**Comment by jeanwsr** [comment]
Probably a mistake of the documentation page. The [example](https://github.com/pyscf/pyscf/blob/master/examples/local_orb/03-split_localization.py) mentioned in that page is correct.

**Comment by tberkel** [comment]
Yes, these were mistakes on the docs. Thanks for finding them. I just submitted a PR.
